Pipfile.lock

```diff
@@ -1,7 +1,20 @@
 {
     "_meta": {
         "hash": {
-            "sha256": "d6131fc6bb32a8212eeebf2348c457d8e2b3ee816f4efb0603666c850ffd8da6"
+            "sha256": "965c019e00fe38782f6b122d2615c37c079098f1084feaab10a760063b171b3f"
+        },
+        "host-environment-markers": {
+            "implementation_name": "cpython",
+            "implementation_version": "3.6.1",
+            "os_name": "posix",
+            "platform_machine": "x86_64",
+            "platform_python_implementation": "CPython",
+            "platform_release": "17.5.0",
+            "platform_system": "Darwin",
+            "platform_version": "Darwin Kernel Version 17.5.0: Fri Apr 13 19:32:32 PDT 2018; root:xnu-4570.51.2~1/RELEASE_X86_64",
+            "python_full_version": "3.6.1",
+            "python_version": "3.6",
+            "sys_platform": "darwin"
```

Comment:
Hmm is that going to be a problem if we run this under Linux in Docker? 🤔

Reply:
Hmm I don't know pipenv well enough to understand the implications of the platform info here.

Reply:
Update pipenv, it removed all this metadata in a more recent version. It was ony for auditing anyway.

Reply:
Looking at https://github.com/pypa/pipenv/commit/ecf78424c1740bae3511c731c032a3768154072c it should be resolved in pipenv 11.0.0 and later.

Reply:
Great! Thanks @davehunt 👍